tensorflow_datasets/image/lost_and_found_test.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for LostAndFound dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import lost_and_found
+from tensorflow_datasets.testing.cityscapes import generate_ids, create_zipfile
+
+
+class LostAndFoundTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = lost_and_found.LostAndFound
+  BUILDER_CONFIG_NAMES_TO_TEST = ['semantic_segmentation', 'full']
+  SPLITS = {
+      "train": 4,  # Number of fake train example
+      "test": 2,  # Number of fake test example
+  }
+  # files as generated by fake data functions below
+  DL_EXTRACT_RESULT = {
+      'image_left': 'leftImg8bit.zip',
+      'image_right': 'rightImg8bit.zip',
+      'disparity_map': 'disparity.zip',
+      'gt': 'gtCoarse.zip'}
+
+
+if __name__ == "__main__":
+  tf.compat.v1.enable_eager_execution()
+
+  # create fake files
+  example_dir = ('tensorflow_datasets/testing/test_data/fake_examples/'
+                 'lost_and_found')
+  testing.test_utils.remake_dir(example_dir)
```

Comment:
You shouldn't create fake examples on every testing. You should just create and add fake examples.

Reply:
@us I did as suggested and added the data to the repository in 06f7bca. However, this adds 120 MiB to the repository. Maybe there is another solution that I am missing? E.g. creating the data as part of the package installation, or only if tests are invoked and the data is not already there?

Reply:
@hermannsblum - Thanks for the PR.

the 120 MB is too much. One way to bring it down: create the fake image that it not a noise but (for example) the simple pattern (even one-color).

This way such files would compress as png lot better.

Reply:
@cyfra Totally agree. I adjusted the random png generation in `fake_data_utils.py` to always generate pngs from a 4x4 random pattern. However, this function is potentially also used by other scripts, so should I add an argument to switch this on and off for backwards compatibility? I don't think huge png files are the wanted behaviour in any case, but I don't knwo how you usually handle this here.

Reply:
Yes, it'd be great to have such argument (probably as default?).
But please do it in separate pull request.